packages/tooltip/test/tooltip-timers.test.js

```diff
@@ -310,7 +324,7 @@ describe('timers', () => {
       mouseleave(targets[0]);
       await aTimeout(1);
 
-      expect(overlays[0].opened).to.be.false;
+      expect(overlays[0].opened).to.be.not.ok;
```

Comment:
This test was previously broken as the tooltip was still open on `mouseenter` due to visibility observer.
Now it's no longer open as expected, so `opened` property is `undefined` and not `false`.